lib/ecdsa.js

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import {webcrypto} from 'node:crypto';
+const {CryptoKey} = webcrypto;
+export {CryptoKey, webcrypto};
+
+export const EcdsaCurve = {
```

Comment:
Do `EcdsaCurve` and `EcdsaHash` need to be somewhere else so they exist when `ecdsa-browser.js` is in use?  Maybe the constants file?  If they move there, then these files could be renamed `crypto{-browser}.js`, or similar, since that's all that would contain.

Reply:
Yes, they most likely do because I noticed that this caused errors in the karma test. I will fix this soon.